usecases/backup/backend.go

```diff
@@ -19,8 +19,10 @@ import (
 	"path"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/semi-technologies/weaviate/entities/backup"
 	"github.com/semi-technologies/weaviate/entities/modulecapabilities"
+	"github.com/semi-technologies/weaviate/usecases/monitoring"
 )
 
```

Comment:
Also I think It makes more sense to have the backup ID as part of the metrics.
This is because as a user I might want to see the metrics of a specific backup.